### DISCUSSION TITLE
vercel --local-config not working

### DISCUSSION THREAD

**Discussion by overshom** [vercel/vercel#9648] [Help] [answered]
I use vercel cli to build app locally and then `vercel deploy --prebuilt` .
`Vercel CLI 28.16.15` - the latest version - still not working. Before this version tried `28.16.12` no luck as well.

For that I want to use command like `vercel --local-config vercel-dev.json build` as per documentation https://vercel.com/docs/cli/global-options#local-config

But it does not work 🐛 
json file ignored 🤷‍♂️ 

I tried other names for json file like `vercel -A vercel/dev/vercel.json build` but still no success at all.

I need to use different buildCommand for different branch deployment
So, my `vercel.json` file looks like that:
```
{
  "buildCommand": "yarn build:dev"
}
```

What am I doing wrong? Why vercel cli cannot pick up specified config file?

P.S. it is not only about different env, but in general about `json` config with all the other options including buildCommand.

**Comment by cb1kenobi** [comment, marked as answer]
I have verified this issue and have opened a pull request for further discussion: https://github.com/vercel/vercel/pull/9675.

**Reply by cb1kenobi** [reply]
This issue has been fixed and shipped in Vercel CLI 28.17.0. Run `npm install -g vercel` to upgrade!

**Reply by overshom** [reply]
Thank you! @cb1kenobi Good job.
